src/MessageSender.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import "./MessageSender.css"
+
+
+function MessageSender(props: {
+    OnAdd: any,
+    ToggleWindow: () => void,
+    WindowState: boolean,
+    TempNew: string,
+    IsNewPostInProcess: boolean
+}) {
+
+
+    const [text, setText] = useState('');
+    useEffect(() => {
+        setText(props.TempNew);
+      });
+
+    // const handleSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
+    //     e.preventDefault();
+    //     props.OnAdd(input);
+    //     setInput("");
+    // }
+
+    const handleClick = (e: React.MouseEvent<HTMLElement>) => {
+        e.preventDefault();
+        document.body.style.overflow = "hidden"
+        // document.getElementsByClassName("create_post_container")[0].setAttribute('style', 'z-index:1')
+        document.getElementsByClassName("opacitySolid")[0].setAttribute('style', 'z-index:0');
```

Comment:
dont... :)

Reply:
Ok :( 
but people really don't do that anymore?

Reply:
Nop, because this operations act directly on the Dom, and react uses the virtual Dom. If you don't understand these concepts, try to read about it, it is important

Reply:
I read about it, not enough, but I do understand the general concept